packages/paste-core/components/button-group/src/ButtonGroup.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {styled, css} from '@twilio-paste/styling-library';
+
+const AttachedStyled = styled.div(
+  css({
+    '& > button:focus': {
```

Comment:
question: can you tell me a little bit about the `'& >` notation? I haven't used it before and I want to understand it more.